src/transformStateWithClones.js

```diff
@@ -6,8 +6,44 @@
  *
  * @return {Object[]}
  */
+/* eslint-disable */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const TYPEPROPERTY = 'addProperties';
+  const TYPEREMOVE = 'removeProperties';
+  const TYPECLEAR = 'clear';
+  const ARRAYRESULT = [];
+  const STATECOPY = Object.assign({}, state);
```

Comment:
```suggestion
  const arrayResult = [];
  const stateCopy = Object.assign({}, state);
```
Use camelCase for default variables